app/lib/utils/addressFormatter.js

```diff
@@ -0,0 +1,16 @@
+const joinTruthyValues = require('./utils').joinTruthyValues;
+
+function addressFormatter(address) {
+  if (!address || Object.keys(address).length === 0) { return undefined; }
+
+  let addressLines = [];
+  if (address.addressLines) {
+    addressLines = address.addressLines;
+  }
```

Comment:
this block could be written as:
`const addressLines = address.addressLines || [];`

Reply:
Nice, thanks. Changes made.